lua/lualine/components/windows/init.lua

```diff
@@ -0,0 +1,75 @@
+local Window = require('lualine.components.windows.window')
+local M = require('lualine.components.buffers'):extend()
+local highlight = require('lualine.highlight')
+
+local default_options = {
+  disabled_filetypes = {},
```

Comment:
Is specifying these two here necessary ? They do get the same value from buffers component like other unspecified option defaults.

Reply:
I think we can remove them, I probably just added them because I was not sure about where those options are defined.
Since I needed them I declared them just to be safe, if they are always present then it's indeed unnecessary.

Reply:
Ohh these are new options you added in windows . I mistakenly thought these were present in buffers component too . It's fine these are suppose to be here.